test/webview/SampleView.test.ts

```diff
@@ -273,6 +289,35 @@ describe('SampleView', () => {
             samplePageHtml.should.contain(`<button class="open-button" onclick="openFile('applications','JavaScript Application')">Open Locally</button>`); // Disabled open button
             samplePageHtml.should.contain(`<button class="open-button" onclick="openFile('applications','TypeScript Application')">Open Locally</button>`); // Disabled open button
         });
+
+        it('should get correct html when cloned and there is no internet connection', async () => {
+            const repositoryRegistryExistsStub: sinon.SinonStub = mySandBox.stub(RepositoryRegistry.prototype, 'exists').returns(true);
```

Comment:
again don't need to stub this just add something to the registry

Reply:
Fixed